plugins/kmeans.cc

```diff
@@ -197,7 +198,15 @@ run(const ProcedureRunConfig & run,
     if (!runProcConf.modelFileUrl.empty()) {
         try {
             Datacratic::makeUriDirectory(runProcConf.modelFileUrl.toString());
-            kmeans.save(runProcConf.modelFileUrl.toString());
+            Json::Value md;
```

Comment:
Why is this inline here? We also lost backward compatibility.  I am not sure if any use kmeans model in production but they should be made aware of this.